guides/source/configuring.md

```diff
@@ -1507,6 +1507,17 @@ The default value depends on the `config.load_defaults` target version:
 | (original)            | `true`               |
 | 7.0                   | `false`              |
 
+#### `config.action_view.prepend_content_exfiltration_prevention`
+
+Determines whether or not the `form_tag` and `button_to` helpers will produce HTML tags prepended with additional matter designed to prevent exfiltration of form data. The default value is `false`.
+
+The default value depends on the `config.load_defaults` target version:
+
+| Starting with version | The default value is |
+| --------------------- | -------------------- |
+| (original)            | `false`              |
+| 7.0                   | `false`              |
```

Comment:
Are we planning to enable this by default in 7.1?  I am all for that, but, if so, this should be `7.1` / `true` instead of `7.0` / `false`, yes?  And we should add an entry to the [Default Values for Target Version 7.1](https://edgeguides.rubyonrails.org/configuring.html#default-values-for-target-version-7-1) subsection.

If not, I'm not sure I see the value of including this table here, since "The default value depends on the `config.load_defaults` target version" is not actually true.

I realize you were just responding to https://github.com/rails/rails/pull/44472#discussion_r810376078, but when a default is not `config.load_defaults`-dependent, we typically just state the default value ([example](https://guides.rubyonrails.org/v7.0/configuring.html#config-action-view-image-loading)).  (In fact, these tables are a [relatively new addition](https://github.com/rails/rails/pull/44038), specifically added for versioned defaults.)